### DISCUSSION TITLE
How to change the yaml prettier formatter to yamlfmt

### DISCUSSION THREAD

**Discussion by nmathew** [LazyVim/LazyVim#3979] [General]
To change this I removed "yaml" from this file https://github.com/LazyVim/LazyVim/blob/11268d8ff1c3675dd4afe45dfd0348cd36a98731/lua/lazyvim/plugins/extras/formatting/prettier.lua#L27

then added 

```
  formatters_by_ft = {
          lua = { "stylua" },
          fish = { "fish_indent" },
          sh = { "shfmt" },
          yaml = { "yamlfmt" },
        },
```

in
https://github.com/LazyVim/LazyVim/blob/11268d8ff1c3675dd4afe45dfd0348cd36a98731/lua/lazyvim/plugins/formatting.lua#L75

I am using LazyVim starter code, is there any way of specifying this in starter config files, instead of editing these above files?

**Comment by dpetka2001** [comment]
All your customizations should go into files under `~/.config/nvim/lua/plugins/` directory. So, you could do
```lua
-- ~/.config/nvim/lua/plugins/format.lua
return {
  {
    "stevearc/conform.nvim",
    opts = {
      formatters_by_ft = {
        yaml = { "yamlfmt" },
      },
    },
  },
}
```

**Reply by dpetka2001** [reply]
It's mentioned in the [docs](https://www.lazyvim.org/configuration), so please do kindly read them through.

**Reply by nmathew** [reply]
Thanks you...
